### PR TITLE
Add info about bugzilla bugs requirements to appear in the list

### PR DESCRIPTION
**Pull request by fiennyangeln**
We need to show that only bugzilla bugs with `good-first-bug` keywords will appear.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,8 +59,8 @@ This will start a local development server on port 5000 (http://localhost:5000).
 
 ## Adding a project
 
-Codetribute can read from both GitHub and Bugzilla. To add a new entry to the site, create a file `<project-name>.yml` 
-in `src/data` using the [template example](#template-example) as the initial setup. 
+Codetribute can read from both GitHub and Bugzilla. Only bugzilla bugs that have `good-first-bug` keyword will appear in the list.
```
Comment (helfi92) [inline README.md]: Codetribute can read from both GitHub and Bugzilla. Regarding the latter, only bugs with the `good-first-bug` keyword will appear in the list.

File: README.md [hunk 2]
```diff
@@ -59,7 +59,7 @@ This will start a local development server on port 5000 (http://localhost:5000).
 
 ## Adding a project
 
-Codetribute can read from both GitHub and Bugzilla. Only bugzilla bugs that have `good-first-bug` keyword will appear in the list.
+Codetribute can read from both GitHub and Bugzilla. Regarding the latter, only bugs with `good-first-bug` keyword will appear in the list.
```
Comment (helfi92) [inline README.md]: s/with `good-first-bug` keyword/with the `good-first-bug` keyword/

**Comment by fiennyangeln** [comment]
I just realized we haven't specidy that `summary` field also get rendered as Markdown

File: README.md [hunk 3]
```diff
@@ -97,7 +97,7 @@ repositories:
 - <Organization Name>/<Repository Name> : <Github Label 2>
 
 ```
-_Note: The `introduction` field is to be rendered as markdown, allowing for bullet points, links and other simple formatting._
+_Note: The `summary` and `introduction` field is to be rendered as markdown, allowing for bullet points, links and other simple formatting._
```
Comment (helfi92) [inline README.md]: `s/field is/fields are/`

**Pull request merged**